buildconfig/pygame-stubs/camera.pyi

```diff
@@ -1,27 +1,23 @@
-from typing import List, Optional, Union, Tuple
+from typing import Optional, Sequence, Tuple, Union
+
 from pygame.surface import Surface
 
+
 def init() -> None: ...
 def quit() -> None: ...
-def list_cameras() -> List[str]: ...
```

Comment:
If it was an input type, I can see the benefit of a Sequence, but not on a return type. Seems to me List is better here?

Reply:
Fixed

Reply:
fixed